Content.Shared/Materials/SharedMaterialReclaimerSystem.cs

```diff
@@ -87,6 +108,15 @@ private void OnActiveUnpaused(EntityUid uid, ActiveMaterialReclaimerComponent co
         component.EndTime += args.PausedTime;
     }
 
+    private void OnAttemptDamageContact(EntityUid uid, MaterialReclaimerComponent component, AttemptDamageContactEvent args)
+    {
+        if (!component.Enabled || !HasComp<EmaggedComponent>(uid))
+        {
+            args.Cancel();
+        }
+        TryStartProcessItem(args.OtherUid, uid, component);
```

Comment:
Is this line necessary? It is already present in OnCollide.

Reply:
It was put in as an attempt to squish the last bug that needs fixing. TLDR, probably doesn't need to be here, no. A lot of this is messy code and will be cleaned if I ever figure out the bug.